src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+
+
+    default void eat() {
+        System.out.println("All animals eat something");
+    }
+
+    static void humans() {
+        System.out.println("Humans are not animals");
+    }
+
+    void test();
+
+    String toString();
```

Comment:
toString() is not acceptable in the interface. It could be here but in classes you're not overriding the toString method from the interface. Remove the toString here and add another method that does something else.

Reply:
I noticed that, Intellij doesn't suggest even to override it. It just a lack of ideas on what method else to add :)
I will figure something out.